pkg/config/server/testdata/scylla-manager.yaml

```diff
@@ -46,3 +46,19 @@ repair:
   long_polling_timeout_seconds: 5
   age_max: 12h
   graceful_stop_timeout: 60s
+
+client:
```

Comment:
`client` is too generic, maybe `agent` since it controls how manager server talks to agent?